buildbreaker.properties

```diff
@@ -0,0 +1,13 @@
+category=Integration
+description=Break the analyzer if the project does not pass its Quality Gate
+homepageUrl=https://github.com/adnovum/sonar-build-breaker
+archivedVersions=
+publicVersions=2.3.1
+defaults.mavenGroupId=org.sonarqubecommunity.buildbreaker
+defaults.mavenArtifactId=sonar-build-breaker-plugin
+
+2.3.1.description=General cleanup & inclusion in SQ Marketplace
+2.3.1.sqVersions=[7.3, LATEST]
```

Comment:
Versions below 7.9 are archived. This range needs to start at 7.9

Reply:
Thanks, I updated the range.